### DISCUSSION TITLE
Incomplete error message

### DISCUSSION THREAD

**Discussion by FangJiangyi** [PaddlePaddle/PaddleSpatial#207] [BaiduKDDCup2022]
A submission error occured as shown in the following figure:
![image](https://user-images.githubusercontent.com/71580867/174708803-ee5add0f-205e-4752-9eff-0d43946ca070.png)
I wonder if I could obtain the whole error message.
Our team name is leyezuTeam, and the submission time is 2022-06-21 10:55.

**Comment by ethanyxx** [comment]
Hi,

Below is the error log related to your submission.
![截屏2022-06-24 17 33 15](https://user-images.githubusercontent.com/62667125/175507950-7f04ea8c-273f-4a3b-9e5c-fd324aaef23b.png)
